sdcm/tester.py

```diff
@@ -389,26 +389,14 @@ def _init_test_duration(self):
     def init_argus_run(self):
         try:
             self.test_config.init_argus_client(self.params)
-            try:
-                status = self.test_config.argus_client().get_status()
-                is_argus_run_exists = True
-                message = f"test_id {self.test_config.test_id()} already exists in Argus with status: {status}"
-            except ArgusClientError as exc:
-                if exc.args[1] == 'DoesNotExist':
-                    message = f"test_id {self.test_config.test_id()} does not exist in Argus"
-                    is_argus_run_exists = False
-                else:
-                    raise
-            self.log.info(message)
-            if not is_argus_run_exists:
-                self.test_config.argus_client().submit_sct_run(
-                    job_name=get_job_name(),
-                    job_url=get_job_url(),
-                    started_by=get_username(),
-                    commit_id=get_git_commit_id(),
-                    sct_config=self.params,
-                )
-                self.log.info("Initialized Argus TestRun with test id %s", self.test_config.argus_client().run_id)
+            self.test_config.argus_client().submit_sct_run(
```

Comment:
we no longer need to verify if argus run exists?

Reply:
Sinse it needs to be called again to make sure we pass the SCT parameters.

Reply:
Affirmative, backend logic now will either create a non-existent run or update the existing one.